crates/types/src/config/worker.rs

```diff
@@ -335,6 +335,10 @@ impl StorageOptions {
     pub fn data_dir(&self) -> PathBuf {
         super::data_dir("db")
     }
+
+    pub fn snapshots_staging_dir(&self) -> PathBuf {
+        super::data_dir("pp-snapshots")
```

Comment:
I noticed that `super::data_dir` is feature gated by `#[cfg(any(test, feature = "test-util"))]` won't this mean it should not be used outside of tests?

Reply:
I hadn't noticed that, just copied what we already do in `data_dir(..)` just above. Doesn't seem to hurt - but I'm somewhat surprised that this works. And we already use that and `node_dir()` from production code elsewhere.

Reply:
yeah, i am surprised that it's working to!